index.bs

```diff
@@ -486,6 +486,69 @@ reasonable future change. We could run [=process a network cookie access for bou
 in the <a spec="fetch">HTTP-network-or-cache fetch</a> algorithm after step 8.21.1.2,
 "... [=append=] (`Cookie`, cookies) to httpRequest’s [=header list=]. ..."
 
+<h5 id="bounce-tracking-mitigations-service-worker-activation-monkey-patch">Service Worker Activation Monkey Patch</h5>
+
+Each [=top-level traversable=] maintains a record of which sites have activated service workers in the current [=extended navigation=].
+
+Insert the following steps in the [activate]((https://w3c.github.io/ServiceWorker/#activate) algorithm before step 11,
+"Run the [Update Worker State](https://w3c.github.io/ServiceWorker/#update-worker-state) algorithm...":
+
+1. Run [=process a service worker activation for bounce tracking mitigations=] given <var ignore>registration</var> and <var ignore>matchedClients</var>.
+
+<div algorithm>
+
+To <dfn>process a service worker activation for bounce tracking mitigations</dfn>
+given a [=service worker registration=] |registration| and a list of [=service worker clients=] |matchedClients|,
+perform the following steps:
+
+1. Let |host| be |registration|'s [=service worker registration/scope url=]'s [=host=].
+1. [=list/For each=] |client| in |matchedClients|:
+    1. Let |browsingContext| be |client|'s [=environment/target browsing context=].
+    1. If |browsingContext| is null, then [=iteration/continue=].
+    1. Let |topLevelTraversable| be |browsingContext|'s [=browsing context/top-level traversable=].
+    1. If |topLevelTraversable|'s [=top-level traversable/bounce tracking record=] is null, then [=iteration/continue=].
+    1. [=set/Append=] |host| to |topLevelTraversable|'s [=top-level traversable/bounce tracking record=]'s
+        [=bounce tracking record/storage access set=].
+
+</div>
+
+<h5 id="bounce-tracking-mitigations-storage-access-monkey-patch">Storage Access Monkey Patch</h5>
+
+Each [=top-level traversable=] maintains a record of which sites have accessed storage in the current [=extended navigation=].
+
+Insert the following steps in the <a spec="storage">obtain a storage bottle map</a> algorithm before step 10, "Return <var ignore>proxyMap</var>":
+
+1. Run [=process a storage access for bounce tracking mitigations=] given <var ignore>environment</var>.
+
+<p class=note>[Note](https://whatwg/storage#165): This patch has to be run whenever a site accesses non-cookie storage.
```

Comment:
I think it's still an issue since we suspect either this patch or the Storage spec has to change. The `Issue(org/repo#nnn)` syntax isn't a link itself; it tells Bikeshed to embed a link: https://speced.github.io/bikeshed/#remote-issues.

```suggestion
Issue(whatwg/storage#165): This patch has to be run whenever a site accesses non-cookie storage.
```

Reply:
Done. Yeah, I figured, but it's annoying that Note doesn't have the same syntax option :)